src/components/App/styledComponents.styles.ts

```diff
@@ -1,5 +1,84 @@
 import styled from 'styled-components';
 
+type StyledCardProps = {
+  contents: string
+}
+
+export const StyledCard = styled.section<StyledCardProps>`
+  transition: all .3s ease-Out;
+  background-color: #ffffff;
+  display: ${props => {
+    if(props.contents === 'empty'){
+      return 'grid;'
+    } else {
+      return 'flex;'
+    }
+  }};
+  flex-direction: column;
+  width: ${props => {
+    if(props.contents === 'wedding') {
+      return '30%;'
+    } else {
+      return '75%;'
+    }
+  }};
+  height: ${props => {
+    if(props.contents === 'wedding') {
+      return '40em;'
+    } else {
+      return '75%;'
+    }
+  }};
+  margin: 1%;
+  border-radius: 30px;
+  box-shadow: 1px 1px 20px #A8826F;
+  font-size: 12px;
+  overflow: ${props => {
+    if(props.contents === 'list') {
+      return 'scroll;'
+    } else {
+      return 'hidden;'
+    }
+  }};
+  align-items: ${props => {
+    if(props.contents === 'wedding') {
+      return 'stretch;'
+    } else {
+      return 'center;'
+    }
+  }};
+  justify-items: ${props => {
+    if(props.contents === 'list') {
+      return 'flex-start;'
+    } else {
+      return 'center;'
+    }
+  }};
+  padding-top: ${props => {
+    if(props.contents === 'wedding') {
+      return '0px;'
+    } else {
+      return '30px;'
+    }
+  }};
+  padding-bottom: ${props => {
+    if(props.contents === 'wedding') {
+      return '0px;'
+    } else {
+      return '30px;'
+    }
+  }};
+  &:hover {
+    box-shadow: ${props => {
+      if (props.contents === 'wedding') {
+        return '1px 1px 20px inset #A8826F;'
+      } else {
+        return '1px 1px 20px #A8826F;'
+      }
+    }};
+  }
+`
+
```

Comment:
So I guess I didn't know enough about styled components. Pretty cool you use methods like that to determine styling